lib/media/catalog/index.ts

```diff
@@ -60,63 +61,68 @@ export function isCatalog(catalog: any): catalog is Catalog {
 }
 
 export interface Track {
-	kind: string
-	container: string
+	name: string
 }
 
 export interface Mp4Track extends Track {
-	container: "mp4"
 	init_track: string
 	data_track: string
 }
 
-export interface AudioTrack extends Track {
-	kind: "audio"
-	codec: string
-	channel_count: number
-	sample_rate: number
-	sample_size: number
+export interface SelectionParamsVideo {
+	codec: string;
+	height: number;
+	width: number;
+	frame_rate: number
 	bit_rate?: number
 }
 
+export interface SelectionParamsAudio {
```

Comment:
Technically, the draft doesn't distinguish between audio/video types but it definitely should.